src/workload.rs

```diff
@@ -516,6 +539,7 @@ impl WorkloadStore {
     fn insert_xds_workload(&mut self, w: XdsWorkload) -> anyhow::Result<()> {
         let workload = Workload::try_from(&w)?;
         let wip = workload.workload_ip;
+        let widentity = workload.identity();
```

Comment:
Forgot something import - we can only do this when workload node == current node (cfg.local_node). Otherwise it will always fail

Reply:
can you followup with a fix?

Reply:
will do